documentation/language/04_operators.md

```diff
@@ -1,11 +1,176 @@
 ---
 id: operators
-title: Leo Operators Reference
-sidebar_label: Operators
+title: Operators and Expressions
```

Comment:
Can I still find the aleo instructions operators anywhere? Did you make an issue on github.com/AleoNet/Welcome?

Reply:
As a quickfix, you could add it in a far-away hidden references section. It's essential for me and other researchers to be able to look up and review.

Reply:
https://developer.aleo.org/guides/aleo/opcodes